site/content/docs/monitoring/_index.md

```diff
@@ -30,11 +30,13 @@ A picture is a thousand words:
 
 1. A cron process picks up a check based on its schedule, say every 5 minutes. It validates that the check is not in progress at the moment to avoid race conditions. The check is put into a queue to be run from the next configured data center location.
 2. If the check is an API check and has a [setup script](/docs/api-checks/setup-teardown-scripts/), the setup script is executed. 
-3. The check is executed. If the check fails and "double check" is enabled, the check is retried once in a different location.
-The other location is picked, at random, from all the configured locations. If only one location has been selected, then the other location is picked at random from all available locations. Setup scripts are executed again during the retry.
+3. The check is executed.
 4. If the check is an API check and has a [teardown script](/docs/api-checks/setup-teardown-scripts/), the teardown script is executed.
 Teardown scripts are run *before* any assertions are validated.
-5. The result is stored in our central database and any alerts are sent where applicable.
+5. The result is stored in our central database.
+6. If the check fails and “double-check” is enabled, the process starts over from a different location. The other location is picked, at random, from all the configured locations. If only one location has been selected, then the other location is picked at random from all available locations. Any setup & teardown scripts are run again as part of the process.
+7. When applicable, alerts are send out in requested channels.
```

Comment:
I think we need to explain here when the whole sequence finishes.
* check passed, or the final attempt was executed
* only then we do send alerts (not for initial attempts)

Reply:
@jan-osch Done! Lemme know if it's good.